GitCommands/Git/GitModule.cs

```diff
@@ -880,7 +880,7 @@ public void RunMergeTool(string? fileName = "", string? customTool = null)
                 { !string.IsNullOrWhiteSpace(fileName), "--" },
                 fileName.ToPosixPath().QuoteNE()
             };
-            using var process = _gitExecutable.Start(args, createWindow: true);
+            using var process = _gitExecutable.Start(args, createWindow: true, throwOnErrorOutput: false);
```

Comment:
In my tests, I could not see any errors from git.
How to reproduce, @gerhardol?

Reply:
You need to set https://git-scm.com/docs/git-mergetool#Documentation/git-mergetool.txt-mergetoollttoolgttrustExitCode
The tool need to set the exit code too. At least Beyond Compare does that, not checked others.

Then you need to make a merge with a conflict and not resolve the conflict, but that part I am sure you follow.

Reply:
Thank you for the explanation.

Reply:
Since `RunMergeTool` does not check the results, I have reverted this from this PR.
If errors must really be ignored, I'd like to handle it in a dedicated PRs on principle.

I have not seen any popups when aborting the merge conflict resolution in BC - with `git config mergetool.bc.trustExitCode true`.